frontend/src/MainPage/OnLoadUIFolder/TextArea.tsx

```diff
@@ -0,0 +1,46 @@
+import { InputBoxArrowLogo } from "../svg/InputBoxArrowLogo";
+import { useState,useContext, useEffect } from "react";
+import { MainPageContext } from "../context/MainPageContext";
+
+
+
+export const TextArea = () => {
+  const [text,setText] = useState("");
+
+  const message = "Where can I find a young, hard working, and passionate developer?";
+  const mainPageContext = useContext(MainPageContext);
+  const {setIsOnLoadUIFinished} = mainPageContext;
+
+  useEffect(() => {
```

Comment:
same as the other animation code, we should probably add a TODO comment here linking to same issue regarding encapsulating this into a reusable custom hook